Content.Server/Revolutionary/Components/Exiled.cs

```diff
@@ -0,0 +1,12 @@
+using Content.Server.Flash;
+using Content.Server.GameTicking.Rules;
+using Robust.Shared.GameStates;
+
+namespace Content.Server.Revolutionary.Components;
```

Comment:
this file is named incorrectly: it should be ExiledComponent (or whatever the class is called)